docs/source/nodes_and_pipelines/modular_pipelines.md

```diff
@@ -1,156 +1,66 @@
-# Modular pipelines
+# Reuse pipelines with namespaces
 
-## What are modular pipelines?
+In many typical Kedro projects, a single (“main”) pipeline increases in complexity as the project evolves. To keep your project fit for purpose, we recommend you separate your code into different pipelines (modules) that are logically isolated and can be reused.
```

Comment:
I don't think this is the purpose of namespace. Even without namespace, users can have multiple pipeline. i.e.

- pipelines
  - data_science
  - data_processing